src/blog/v8-release-85.md

```diff
@@ -0,0 +1,126 @@
+---
+title: 'V8 release v8.5'
+author: 'Sathya Gunasekaran'
+avatars:
+date: 2020-07-21
+tags:
+ - release
+description: 'V8 release v8.5'
```

Comment:
This should include a short description of features, see other posts for examples.

Reply:
Changed it